src/graph/GraphViewsManager.ts

```diff
@@ -37,7 +34,7 @@ export class GraphViewsManager implements IServerProvider {
     config: GraphConfiguration
   ): Promise<void> {
     try {
-      var [id, server] = await this.getOrCreateServer(config);
+      var [id,] = await this.getOrCreateServer(config);
```

Comment:
nit: You could change the signature of getOrCreateServer to return only id

Reply:
I'd be fine with that.